.coveragerc

```diff
@@ -708,6 +708,17 @@ omit =
     homeassistant/components/repetier/__init__.py
     homeassistant/components/repetier/sensor.py
     homeassistant/components/remote_rpi_gpio/*
+    homeassistant/components/renault/__init__.py
+    homeassistant/components/renault/binary_sensor.py
+    homeassistant/components/renault/climate.py
+    homeassistant/components/renault/config_flow.py
```

Comment:
The config flow should not be excluded from testing. We require configuration flows to be tested with a 100% test coverage.

Reply:
Config flow tests added in fb46b96

Reply:
```suggestion
```

Reply:
Thanks @Quentame, should be fixed in cc24467